src/app/modules/item/task-communication/rxjschannel.ts

```diff
@@ -50,13 +52,15 @@ export class RxMessagingChannel {
         result
           .pipe(take(1))
           .subscribe({
-            next: data => transaction.complete(data),
+            next: data => {
+              if (!this.destroyed) transaction.complete(data);
```

Comment:
We could even save the subscription and unsubscribe on "destroy" so that there is not a piece of code still waiting on something that does not make sense anymore. 
At first, it looks a bit like  `if (!this.destroyed)` is enough for safety.... but just reading the class (so at least in the scope of this class), I don't see the guaranty that `result` always emits... with the risk of retaining the submission (and so `this` as well).  -> I am not 100% of what I am writing here ;-)
(and so, if we do the unsubscription, probably `if (!this.destroyed)` is not useful anymore)

Reply:
For instance here we are saving a grade (xhr) and then notifying the task that the operation went OK (the example works with other xhr requests).
Unsubscribing would mean cancelling the saving grade xhr request when the task is destroyed and I don't think that's what we want, is it?
To me what we want is: 1) if task is active, notify it that the operation went OK, and 2) if the task is destroyed, don't notify it that it went ok.
And personally I don't see which observable might never end in this context since here we use observables that emit a single value (`take(1)`). The only risk is an observable that would never completes or fall in error state, and even that I don't know where it would be declared :thinking: 

By writing that, I realise that I have handle only success use case and not failure, I'll add a fix.

Reply:
> To me what we want is: 1) if task is active, notify it that the operation went OK, and 2) if the task is destroyed, don't notify it that it went ok.

Makes sense! Agreed.

>  And personally I don't see which observable might never end in this context since here we use observables that emit a single value (take(1)). The only risk is an observable that would never completes or fall in error state, and even that I don't know where it would be declared 🤔

So because you do assumptions on the way caller use this function... of course if it xhr only.
Maybe you should add this in the `bind` spec.  We should also add a (large) timeout, but that may be overkill.